generator/src/lib.rs

```diff
@@ -1080,7 +1092,11 @@ impl<'a> ConstantExt for ExtensionConstant<'a> {
         self.constant.clone()
     }
     fn variant_ident(&self, enum_name: &str) -> Ident {
-        variant_ident(enum_name, self.name)
+        variant_ident(
+            enum_name,
+            self.name,
+            self.notation == Some("Backwards-compatible alias containing a typo"),
```

Comment:
The string constant appears four times.
A Java coder would factorize it into a private static constant.

Reply:
Fortunately I'm completely oblivious to Java (coders), nor do I think anyone cares for a generator that's on the brink of being rewritten.